.coveragerc

```diff
@@ -0,0 +1,16 @@
+#
+# .coveragerc to control coverage.py
+#
+
+[run]
+branch = True
+omit = 
+    setup.py
+    iris_grib/tests/*
```

Comment:
Yes, really. The point of testing coverage is to determine the percentage of code that is tested. 

By running the tests, coverage calculates the coverage of code that is exercised by running the tests. We don't want to dilute that statistic with the actual test code itself.